ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts

```diff
@@ -30,7 +30,7 @@ class FakeAddressInputComponent {
   private _readonly = false;
   @Output()valueChanged = new EventEmitter();
   @Input()addressErrors;
-  @Input()country;
+  @Input()country = null;
```

Comment:
This is one change for the test. Test was expecting null but we were getting undefined.